meta-sokol-flex-support/recipes-core/images/flex-image.inc

```diff
@@ -13,6 +13,7 @@ LICENSE = "MIT"
 
 inherit core-image
 
-IMAGE_FEATURES:append:sokol-flex = " splash ${@bb.utils.contains('COMBINED_FEATURES', 'alsa', ' hwcodecs tools-audio', '', d)}"
+IMAGE_FEATURES:append:sokol-flex = " splash ${@bb.utils.contains('COMBINED_FEATURES', 'alsa', ' tools-audio', '', d)}"
 IMAGE_INSTALL:append:sokol-flex = " util-linux-mkfs connman"
 IMAGE_INSTALL:append:sokol-flex = " ${@bb.utils.contains('BBFILE_COLLECTIONS', 'openembedded-layer', 'haveged' if not any_incompatible(d, ['haveged'], 'GPL-3.0-only') else '', '', d)}"
+IMAGE_INSTALL:append:sokol-flex = " ${@bb.utils.contains_any('IMAGE_FEATURES', ['multimedia', 'graphics'], ' ${MACHINE_HWCODECS}', '', d)}"
```

Comment:
We don't need a space here before MACHINE_HWCODECS we've the needed space at the start of bb.utils.contains already.

Reply:
Ok thanks @abelal I have removed the space.